### PR TITLE
Update 8b53cdec-d5c9-4c28-af11-611f95c37b34.png

### PR DESCRIPTION
**Pull request by msbemba**
Changed screenshot as the options have changed

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/8191

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/enable-archive-mailboxes.md [hunk 1]
```diff
@@ -48,15 +48,14 @@ If you don't see the **Archive** page in the Microsoft 365 compliance center, as
    > [!NOTE]
    > The **Archive** page shows a maximum of 500 users.
 
-3. In the list of mailboxes, select the user that you want to enable the archive mailbox for.
+3. In the list of mailboxes, select the user that you want to enable the archive mailbox for and clck on **Enable Archive**.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/enable-archive-mailboxes.md]:
```suggestion
3. In the list of mailboxes, select the user that you want to enable the archive mailbox for and select **Enable Archive**.
```

File: microsoft-365/compliance/enable-archive-mailboxes.md [hunk 2]
```diff
@@ -48,15 +48,14 @@ If you don't see the **Archive** page in the Microsoft 365 compliance center, as
...
 
    ![Click Enable in the details pane of the selected user to enable the archive mailbox.](../media/8b53cdec-d5c9-4c28-af11-611f95c37b34.png)
 
-4. In the details pane for the selected user, click **Enable**.
 
    A warning is displayed saying that if you enable the archive mailbox, items in the user's mailbox that are older than the archiving policy assigned to the mailbox will be moved to the new archive mailbox. The default archive policy that is part of the retention policy assigned to Exchange Online mailboxes moves items to the archive mailbox two years after the date the item was delivered to the mailbox or created by the user. For more information, see the **More info** section in this article.
 
-5. Click **Yes** to enable the archive mailbox.
+5. Click **Enable** to enable the archive mailbox.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/enable-archive-mailboxes.md]:
```suggestion
5. Select **Enable** to enable the archive mailbox.
```

File: microsoft-365/compliance/enable-archive-mailboxes.md [hunk 3]
```diff
@@ -80,13 +79,12 @@ To disable an archive mailbox:
    > [!NOTE]
    > The **Archive** page shows a maximum of 500 users.
 
-3. In the list of mailboxes, select the user that you want to disable the archive mailbox for.
+3. In the list of mailboxes, select the user that you want to disable the archive mailbox for and clikc on **Disable Archive**
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/enable-archive-mailboxes.md]:
```suggestion
3. In the list of mailboxes, select the user that you want to disable the archive mailbox for and select **Disable Archive**.
```

File: microsoft-365/compliance/enable-archive-mailboxes.md [hunk 4]
```diff
@@ -80,13 +79,12 @@ To disable an archive mailbox:
...
 
-4. In the details pane, click **Disable**.
 
    A warning message is displayed saying that you'll have 30 days to re-enable the archive mailbox, and that after 30 days, all information in the archive will be permanently deleted.
 
-5. Click **Yes** to disable the archive mailbox.
+5. Click **Disable** to disable the archive mailbox.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/enable-archive-mailboxes.md]:
```suggestion
5. Select **Disable** to disable the archive mailbox.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@tiburd Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Pull request merged**